osf/utils/tokens/__init__.py

```diff
@@ -14,14 +14,14 @@
 class TokenHandler:
 
     HANDLERS = {
-        'approve_registration_approval': functools.partial(handlers.sanction_handler, 'registration', 'approve'),
-        'reject_registration_approval': functools.partial(handlers.sanction_handler, 'registration', 'reject'),
-        'approve_embargo': functools.partial(handlers.sanction_handler, 'embargo', 'approve'),
-        'reject_embargo': functools.partial(handlers.sanction_handler, 'embargo', 'reject'),
-        'approve_embargo_termination_approval': functools.partial(handlers.sanction_handler, 'embargo_termination_approval', 'approve'),
-        'reject_embargo_termination_approval': functools.partial(handlers.sanction_handler, 'embargo_termination_approval', 'reject'),
-        'approve_retraction': functools.partial(handlers.sanction_handler, 'retraction', 'approve'),
-        'reject_retraction': functools.partial(handlers.sanction_handler, 'retraction', 'reject')
+        'approve_registration_approval': functools.partial(handlers.sanction_handler_flask, 'registration', 'approve'),
```

Comment:
This change is so these can still use the same flask auth decorator. The original name is now the unwrapped function inside that decorator. This is so `sanction_handler` can be shared by Django and flask request types.